iroha_schema/src/lib.rs

```diff
@@ -173,6 +180,20 @@ macro_rules! impl_schema_int {
 
 impl_schema_int!(u128, u64, u32, u16, u8, i128, i64, i32, i16, i8,);
 
+impl<I: IntoSchema, P> IntoSchema for FixedPoint<I, P> {
+    fn type_name() -> String {
+        format!("FixedPoint<{}>", I::type_name())
+    }
+    fn schema(metamap: &mut MetaMap) {
+        let _ = metamap
+            .entry(Self::type_name())
+            .or_insert_with(|| Metadata::FixedPoint(FixedMeta { ty: I::type_name() }));
```

Comment:
Also isn't it important to record `P` for the client to understand the number of decimal places?

Reply:
You right. Will update it

Reply:
done